tor/DOCS.md

```
@@ -219,38 +220,43 @@ bridges:
+However, with the `stealth` option enabled, the client needs extra
 configuration to be able to connect.
 
-Add the authentication cookie to your `torrc` client configuration on your
-laptop or mobile device. It would look like this:
+The app generates a key pair for each configured client. The private key file
+is stored at `/ssl/tor/hidden_service/clients/<clientname>.auth_private` and
+contains a line in this format:
 
-```bash
-HidServAuth abcdef1234567890.onion adEG02FAsdq/GAFeNSeLvc haremote1
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)